tensorboard/backend/event_processing/plugin_event_multiplexer_test.py

```diff
@@ -23,9 +23,10 @@
 
 import tensorflow as tf
 
-from tensorboard.backend.event_processing import plugin_event_accumulator as event_accumulator  # pylint: disable=line-too-long
-from tensorboard.backend.event_processing import plugin_event_multiplexer as event_multiplexer  # pylint: disable=line-too-long
-
+from tensorboard.backend.event_processing import plugin_event_accumulator as ea  # pylint: disable=line-too-long
```

Comment:
I'd rather not change the history of this file to make it more succinct. I'm OK with it remaining verbose.